### PR TITLE
chore(seeder): added additional circles and permissions from prod

### PR DESCRIPTION
**Pull request by WikiRik**

### CODE REVIEW

File: scripts/seed.js [hunk 1]
```diff
@@ -120,12 +190,10 @@ async function createCircles() {
 
         await body.update({ shadow_circle_id: membersCircle.id });
 
-        circles.board.push(boardCircle);
         circles.members.push(membersCircle);
 
         if (body.type === 'antenna') {
             circles.antennaMembersCircle = membersCircle;
-            circles.antennaBoardCircle = boardCircle;
         }
```
Comment (HassanCehef) [inline scripts/seed.js:197]: `antennaMembersCircle` is unused. You can remove the whole if block.

**Review by HassanCehef — COMMENTED** [review]
change makes sense.

File: scripts/seed.js (hunk 1 above)
Comment (WikiRik) [reply scripts/seed.js:197]: Indeed, I never looked at that. Are you not using it in your updates to the seeder?

Comment (HassanCehef) [reply scripts/seed.js:197]: nope, I needed to deal with the other one, but you automated the creation of it (and generalized it too). Happy to see that block gone;

**Review by HassanCehef — APPROVED** [review]
🚀

**Pull request merged**